### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by samuel-allan** [mixmatch/ssh-electron#1]
I really love the concept, and it doesn't look like there's anything else like this out there, so, is this project still alive/active? Regardless of whether it is or isn't, I would gladly offer my help if it is needed.

**Comment by samuel-allan** [comment]
I am currently working on a clone over at samuel-allan/ssh-electron. Let me know if development of this repository is halted for some reason.

**Comment by dan-petty** [comment]
Hey, i'd love to look at this project again, especially if you are interested in contributing. I imagine that some of the dependent packages have been updated or there are better npm packages and such to make this perform better...

**Comment by samuel-allan** [comment]
I am incredibly interested, and there are a lot of great options out there (such as ssh2) which would work great with it. It's still questionable whether a full-on node approach should be used (the cleaner way), or plugging in with some C++ for the speed of communication over SSH (the dirtier, but more performant way). I would def. go for pure-node. By the way, do take a look at the progress on my repository so far!

**Comment by dandv** [comment]
@samuel-allan: just a tip, it'd be easier for people to look at [your project](https://github.com/samuel-allan/ssh-electron) if you edited [the comment](https://github.com/mixmatch/ssh-electron/issues/1#issuecomment-283464741) to include a link to it! Seems like nobody has, so far, gone past [the PITA](https://wiki.dandascalescu.com/essays/pita-threshold) of manually entering the URL.

**Comment by dan-petty** [comment]
This project is still in my sights, but I would like to rebase the code off of 
https://github.com/chentsulin/electron-react-boilerplate

and build the UI out with React. I also need to update the npm dependencies.

**Comment by samuel-allan** [comment]
Good idea, I had another resource (listed on my repo) for the actual SSH
backend
